### PR TITLE
Ian/patch/replacing place holder in home page

### PR DESCRIPTION
**Pull request by abc-1211**
-Replace placeholder in home-project to DB data
-Relocating data in account dashboard to parent so as to feed multiple child

### CODE REVIEW

File: src/components/ProjectsList/ProjectsList.js [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
+  }, []);
+
   return (
     <div className="projectList">
       <List className="list">
-        {projects.map((project) => (
-          <a href={"project/" + project._id}>
-            <LinkItem className="listItem btn btn-primary" key={project._id}>
+        {projects.map((project, index) => (
+          <a href={"project/" + project._id} key={index}>
```
Comment (JoeDodgson) [inline src/components/ProjectsList/ProjectsList.js:32]: Change key to the project._id

File: src/components/ProjectsList/ProjectsList.js [hunk 2]
```diff
@@ -1,14 +1,36 @@
...
+export default function ProjectList() {
+
+  const [projects, setProjects] = useState([]);
+
+  
+  const getProject = async() => {
+    try {
+      const dataReturn = await projectService.getAllProjects();
+      const arrayData = dataReturn.data.projectsData;
+      console.log(arrayData);
```
Comment (JoeDodgson) [inline src/components/ProjectsList/ProjectsList.js:15]: Remove console log

File: src/components/ProjectsList/ProjectsList.js [hunk 3]
```diff
@@ -1,14 +1,36 @@
...
+
+      setProjects(arrayData);
+    }
+    catch (err) {
+      console.log(err);
```
Comment (JoeDodgson) [inline src/components/ProjectsList/ProjectsList.js:20]: Correct the format of error console log:
```
console.log(`Error - ProjectsList.js - getProject() - ${error}`)
```

File: src/components/ProjectsList/ProjectsList.js [hunk 4]
```diff
@@ -1,14 +1,36 @@
...
-const projects = [ {_id: "234iuhowef9803rh", title: "Big Ben" }, {_id: "584pduablf9571qt", title: "London Bridge" } ]
-export default function ProjectList(props) {
+import projectService from "../../Services/ProjectService";
+
+export default function ProjectList() {
+
+  const [projects, setProjects] = useState([]);
+
+  
+  const getProject = async() => {
```
Comment (JoeDodgson) [inline src/components/ProjectsList/ProjectsList.js:11]: Leave a comment next to this, stating that this needs to be changed to "getProjectByUserId"

File: src/components/pages/Home/Home.js [hunk 5]
```diff
@@ -1,20 +1,41 @@
...
 import "./styles/Home.css";
 
 function Home(props) {
+
+  const [user, setUser] = useState();
+  const [project, setProject] = useState(null);
+  const getData = async () => {
+    try {
+      const {data} = await AuthService.getInfo("5f5a1ef56157b937082c4229");
+      console.log(data)
```
Comment (JoeDodgson) [inline src/components/pages/Home/Home.js:18]: Remove console log

File: src/components/pages/Home/Home.js [hunk 6]
```diff
@@ -1,20 +1,41 @@
...
+      setUser(data.firstName + " " + data.lastName);
+      setProject(data.project.length);
+    }
+    catch (err) {
+      console.log(`Error - AccountDashboard.js - ${err}`);
```
Comment (JoeDodgson) [inline src/components/pages/Home/Home.js:23]: Amend to:
console.log(`Error - AccountDashboard.js - getData() - ${err}`);

File: src/components/pages/Home/Home.js [hunk 7]
```diff
@@ -1,20 +1,41 @@
...
+
   return (
-    <>
+    <React.Fragment>
       <Navbar />
       <Tabs defaultActiveKey="home" id="uncontrolled-tab-example">
         {/* eventkey is acting as the event listener the tabs won't change unless eventkey is specified but it doesn't need to be the same as the title */}
         <Tab eventKey="home" title="Dashboard">
-          <AccountDashboard />
+          <AccountDashboard  username={user} project={project} />
```
Comment (JoeDodgson) [inline src/components/pages/Home/Home.js:38]: This does not solve the problem of not being able to get user _id. We are still using a static user id to get this user information. Remove these props, and remove use of props in AccountDashboard component

**Review by JoeDodgson — CHANGES_REQUESTED** [review]

File: src/components/ProjectsList/ProjectsList.js (hunk 4 above)
Comment (abc-1211) [reply src/components/ProjectsList/ProjectsList.js:11]: Negative, i think you have looked into the wrong spot, ProjectsList is functioning properly and this does not require any id.

The method i do is to **FIND ALL** the project inside the project collections thats why no specify is required

**Pull request closed without merging**